.circleci/config.yml

```diff
@@ -713,8 +714,9 @@ jobs:
                -e "AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}" \
                -e "AWS_ACCOUNT_ID=${AWS_ACCOUNT_ID}" \
                -e "AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}" \
+               -e "DEPLOYING_COLOR=${DEPLOYING_COLOR}" \
                -e "ENV=${ENV}" \
-               --rm efcms /bin/sh -c "cd web-api && node configure-dynamo-stream.js ${DESTINATION_TABLE} false"
+               --rm efcms /bin/sh -c "./web-api/disable-deploying-dynamo-stream.sh"
```

Comment:
Since it caused some confusion at the start, I suggest changing the name to be more descriptive: `disable-deploying-dynamo-stream-trigger.sh`